frontend/cypress/support/commands.ts

```diff
@@ -36,10 +36,10 @@ Cypress.Commands.add('login', () => {
   cy.getByDataTest('landing-button__bceid').click();
   cy.get('#bceidLogo').should('be.visible');
   cy.get('input[name=user]')
-        .clear()
-        .type(USERNAME, { delay: 50 });
+    .clear()
+    .type(USERNAME, { delay: 50 });
```

Comment:
Can you please update the `README.md` file with the correct name of the cypress environment variables?